docs/tutorial.rst

```diff
@@ -98,6 +98,74 @@ before copying the data over:
 Note that to ensure consistency of operations, the ``pgcopydb snapshot``
 command has been used. See :ref:`resuming_operations` for details.
 
+Use pgcopydb to migrate from a Citus database to a Citus database
+-----------------------------------------------------------------
+
+It is possible to use the previous tutorial to implement support for
+migrating a Citus database to its new hosting environment. For that, the
+``schema-changes.sql`` script from the previous section needs to be the
+Citus script that calls into the `Citus DDL`__ functions.
+
+__ https://docs.citusdata.com/en/v6.0/dist_tables/ddl.html
```

Comment:
```suggestion
__ https://docs.citusdata.com/en/latest/develop/reference_ddl.html
```

6.0 is really old. It is the version that introduced PG9.6 support.

Reply:
The word `develop` in the link I shared may look like it is a documentation page for a development build. It is not. This is in my opinion a poor choice of name for a documentation section: https://github.com/citusdata/citus_docs/tree/master/develop